server/api/core/core.js

```diff
@@ -60,7 +60,10 @@ export default {
     const registeredPackage = this.Packages[packageInfo.name] = packageInfo;
     return registeredPackage;
   },
+  defaultCustomerRoles: [ "guest", "account/profile", "product", "tag", "index", "cart/checkout", "cart/completed"],
```

Comment:
That's right.. I remember now, what triggered the question was the move and seeing them as additions.